LaTeX/TeX.sublime-syntax

```diff
@@ -111,6 +118,46 @@ contexts:
       captures:
         1: punctuation.definition.backslash.tex
 
+  registers:
+    - match: (\\)({{registers}}){{endcs}}
+      captures:
+        1: punctuation.definition.backslash.tex
+        2: storage.type.tex
+      push:
+        - meta_scope: variable.language.tex
+        - include: register-id
+
+  register-id:
+    - match: \d+
+      scope: constant.numeric.tex
+      pop: 1
+    - match: (\\){{cmdname}}
+      captures:
+        1: punctuation.definition.backslash.tex
+      pop: 1
+    - include: paragraph-pop
+    - include: else-pop
+
+  register-allocations:
+    - match: (\\)new{{registers}}{{endcs}}
+      scope: keyword.declaration.register.tex storage.modifier.register.tex
+      captures:
+        1: punctuation.definition.backslash.tex
+      push: def-registername
+
+  def-registername:
+    - match: (?=\\)
+      set:
+        - match: (\\){{cmdname}}
+          scope: entity.name.constant.tex
+          captures:
+            1: punctuation.definition.backslash.tex
+          pop: 1
+        - include: paragraph-pop
+        - include: else-pop
```

Comment:
This context is of the same category as `register-id`. The lookahead `(?=\\)` is not required in this situation. The reason I used it in LaTeX's `newcommand-definition` context is the `{` which also needed to be handled.

```suggestion
    - match: (\\){{cmdname}}
      scope: entity.name.constant.tex
      captures:
        1: punctuation.definition.backslash.tex
      pop: 1
```

Reply:
Good observation about the braces! However, I've just checked, and it is also possible to put the braces there in this case. So I'll have to adapt this and add some tests :)